src/app/components/App/Form.tsx

```diff
@@ -144,6 +160,14 @@ const Form: React.FC = () => {
                   onBlur={handleBlur}
                 />
               </div>
+              <Dropzone setFileSelected={setFileSelected} setImagePreview={setImagePreview}/>
+              <div css={inputStyle}>
```

Comment:
as you have own Dropzone component, would be nice moving this part into it.
so implementation is easier and not get duplicated